enterprise/server/util/oci/oci.go

```diff
@@ -92,11 +93,7 @@ func CredentialsFromProperties(props *platform.Properties) (Credentials, error)
 
 	// If no credentials were provided, fallback to any specified by
 	// --executor.container_registries.
-	if len(*registries) == 0 {
```

Comment:
Why do you not need to look at the registries anymore?

Reply:
We still look at the registries, this just removes the short-circuit if there are no registries configured, so that we can fall back to using the default keychain below.